content/docs/user-guide/components/reference/atom/light.md

```diff
@@ -26,7 +26,7 @@ Quad
 : Emits light from the surface of a rectangle in 3D space. The Quad light type is most useful for illuminating a larger area with diffuse light. By default, the Quad light type uses linearly transformed cosines to calculate accurate lighting. It also supports a fast approximation calculation that's more performant but produces lower-quality light. The Quad light type can emit light from one or both directions.
 
 Polygon   
-: Emits light from an arbitrarily shaped polygon in 3D space. This polygon can have up to 64 points, but the Polygon light type becomes more expensive as the number of points in the polygon increases. The Polygon light type is the most expensive light type, but it produces very realistic area lighting. The Polygon light type can emit light from one or both directions.
+: Emits light from an arbitrarily shaped polygon in 3D space that are approximated by linearly transformed cosines. This polygon can have up to 64 points, but the Polygon light type becomes more expensive as the number of points in the polygon increases. The Polygon light type is the most expensive light type, but it produces very realistic area lighting. The Polygon light type can emit light from one or both directions.
```

Comment:
"Polygon" is singular.
```suggestion
: Emits light from an arbitrarily shaped polygon in 3D space that is approximated by linearly transformed cosines. This polygon can have up to 64 points, but the Polygon light type becomes more expensive as the number of points in the polygon increases. The Polygon light type is the most expensive light type, but it produces very realistic area lighting. The Polygon light type can emit light from one or both directions.
```

Reply:
It's also unclear to me which noun "that is" modifies. Is it "light", "polygon", or "3D space"? Might want to revise slightly for clarity.